enterprise/migrations/0168_auto_20230222_1621.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 3.2.17 on 2023-02-21 18:35
+
+from django.db import migrations, models
+import uuid
+
+
+def create_license_enrollment_uuid(apps, schema_editor):
+    licensed_enrollments = apps.get_model('enterprise', 'LicensedEnterpriseCourseEnrollment')
+    for licensed_enrollment in licensed_enrollments.objects.all():
+        licensed_enrollment.uuid = uuid.uuid4()
+        licensed_enrollment.save()
+
+
+def create_hist_learner_credit_enrollment_uuid(apps, schema_editor):
+    hist_learner_credit_enrollments = apps.get_model('enterprise', 'historicallearnercreditenterprisecourseenrollment')
+    for hist_learner_credit_enrollment in hist_learner_credit_enrollments.objects.all():
+        hist_learner_credit_enrollment.uuid = uuid.uuid4()
+        hist_learner_credit_enrollment.save()
+
+
+def create_hist_license_enrollment_uuid(apps, schema_editor):
+    hist_licensed_enrollments = apps.get_model('enterprise', 'historicallicensedenterprisecourseenrollment')
+    for hist_licensed_enrollment in hist_licensed_enrollments.objects.all():
+        hist_licensed_enrollment.uuid = uuid.uuid4()
+        hist_licensed_enrollment.save()
```

Comment:
I don't think this is what you want - these uuids would be different from the actual models' `uuid` values.
But also, you shouldn't have to directly interact with the historical models - that should come for free when you do `model.save()` in the non-historical models.